Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:alpine3.18 as builder
+FROM rust:1.82-alpine AS builder
```

Comment:
Could go with `1-alpine` or just `alpine` to avoid having to update it in the near future again due to using an outdated rust version.